initfiles/componentfiles/thor/run_thor

```diff
@@ -22,46 +22,46 @@ thorpid=0
 
 RUN_THOR_PID_NAME="$PID/run_thor.`basename $PWD`.pid"
 # prevent two thors starting together
-if [ -e $RUN_THOR_PID_NAME ]; then
-  oldpid=`cat $RUN_THOR_PID_NAME`
-  while ps h $oldpid ; do 
-     echo waiting for process $oldpid to finish
-     sleep 5
-  done
+if [[ -e $RUN_THOR_PID_NAME ]]; then
+    oldpid=$(cat $RUN_THOR_PID_NAME)
+    while ps h $oldpid ; do
+        echo "waiting for process $oldpid to finish"
+        sleep 5
+    done
```

Comment:
Minor: slightly easier to review, if cosmetic space/formatting changes are split in to a separate commit (but same PR).